books/RayTracingTheNextWeek.html

```diff
@@ -1818,25 +1818,27 @@
         ...
         private:
             ...
-            inline double perlin_interp(vec3 c[2][2][2], double u, double v, double w) const {
+            static double perlin_interp(vec3 c[2][2][2], double u, double v, double w) {
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
                 auto uu = u*u*(3-2*u);
                 auto vv = v*v*(3-2*v);
                 auto ww = w*w*(3-2*w);
                 auto accum = 0.0;
-
+                
                 for (int i=0; i < 2; i++)
-                    for (int j=0; j < 2; j++)
-                        for (int k=0; k < 2; k++) {
-                            vec3 weight_v(u-i, v-j, w-k);
-                            accum += (i*uu + (1-i)*(1-uu))
-                                   * (j*vv + (1-j)*(1-vv))
-                                   * (k*ww + (1-k)*(1-ww))
-                                   * dot(c[i][j][k], weight_v);
-                        }
-
+                for (int j=0; j < 2; j++)
+                for (int k=0; k < 2; k++) {
+                    vec3 weight_v(u-i, v-j, w-k);
+                    accum += (i*uu + (1-i)*(1-uu))
+                    * (j*vv + (1-j)*(1-vv))
+                    * (k*ww + (1-k)*(1-ww))
+                    * dot(c[i][j][k], weight_v);
+                }
+                
```

Comment:
Indentation corrupted.

Reply:
Sorry about that, thanks for picking it up